extension/js/common/platform/store/contact-store.ts

```diff
@@ -459,7 +462,7 @@ export class ContactStore extends AbstractStore {
   }
 
   private static dbContactInternalGetOne = async (db: IDBDatabase, emailOrLongid: string): Promise<Contact | undefined> => {
-    if (!/^[A-F0-9]{16}$/.test(emailOrLongid)) { // email
+    if (emailOrLongid.includes('@')) { // email
```

Comment:
We could probably separate this in another PR - no need to have a single ambiguous method. We could have getByEmail and `getByFingerprint` and `getByLongid`